{
	"name": "CVE-2023-26151",
	"seq": "2023-26151",
	"type": "CAN",
	"status": "Candidate",
	"phase": {
		"text": "Assigned",
		"date": "20230220"
	},
	"desc": "Versions of the package asyncua before 0.9.96 are vulnerable to Denial of Service (DoS) such that an attacker can send a malformed packet and as a result, the server will enter into an infinite loop and consume excessive memory.",
	"refs": [
		{
			"text": "https://gist.github.com/artfire52/1540b234350795e0ecb4d672608dbec8",
			"source": "MISC",
			"url": "https://gist.github.com/artfire52/1540b234350795e0ecb4d672608dbec8"
		},
		{
			"text": "https://github.com/FreeOpcUa/opcua-asyncio/commit/f6603daa34a93a658f0e176cb0b9ee5a6643b262",
			"source": "MISC",
			"url": "https://github.com/FreeOpcUa/opcua-asyncio/commit/f6603daa34a93a658f0e176cb0b9ee5a6643b262"
		},
		{
			"text": "https://github.com/FreeOpcUa/opcua-asyncio/issues/1013",
			"source": "MISC",
			"url": "https://github.com/FreeOpcUa/opcua-asyncio/issues/1013"
		},
		{
			"text": "https://github.com/FreeOpcUa/opcua-asyncio/pull/1039",
			"source": "MISC",
			"url": "https://github.com/FreeOpcUa/opcua-asyncio/pull/1039"
		},
		{
			"text": "https://github.com/FreeOpcUa/opcua-asyncio/releases/tag/v0.9.96",
			"source": "MISC",
			"url": "https://github.com/FreeOpcUa/opcua-asyncio/releases/tag/v0.9.96"
		},
		{
			"text": "https://security.snyk.io/vuln/SNYK-PYTHON-ASYNCUA-5673709",
			"source": "MISC",
			"url": "https://security.snyk.io/vuln/SNYK-PYTHON-ASYNCUA-5673709"
		}
	],
	"votes": {}
}
